### ISSUE TITLE
`vue-docgen-api` cannot parse slot definition inside slot usage

### ISSUE THREAD

**Issue by pavelda2** [vue-styleguidist/vue-styleguidist#470]
**Current behavior**

Documented slot definition (`<slot name="checkboxGroupLabel"/>`) is not parsed by vue-docgen-api when it is inside other slot usage (`<template #checkboxLabel>...</template> `)

**To reproduce**

```vue
<app-checkbox v-for="option in options" :key="option">
   <template #checkboxLabel>
        <!-- @slot Slot documentation not parsed -->
        <slot name="checkboxGroupLabel">{{option}}</slot>
   </template>
</app-checkbox>
```

See this example in repository: https://github.com/pavelda2/vue-docgen-api-bug/blob/master/src/components/CheckboxGroup.vue

**Expected behavior**

Library vue-docgen-api should parse slot documentation and therefore vue-styleguidist display the `checkboxGroupLabel` slot documentation.

**Comment by elevatebart** [comment]
I believe this was fixed in 3.16.2 can you update and see ?

**Comment by elevatebart** [comment]
@pavelda2 please re-open this issue if you find the bug in a version over 3.16.2.
It should not happen though.

**Comment by pavelda2** [comment]
Great. I confirm that the bug was fixed in 3.16.2 👍.
